### DISCUSSION TITLE
Merging of TripCalculator into development branch

### DISCUSSION THREAD

**Discussion by MarkW06071969** [MarkW06071969/Tafe202206#20] [General]
Hi all,

Stephen and I have now merged our functions into the 'development' branch of the GitHub Repository.

Have just had a talk with KT:  if Hamish (hope you get better soon) could clone the GitHub repository on
his own machine and  then merge the TripCalculator branch with the development branch and then push it back
to the GitHub Repository that would be great if that could be done soon.

If there any questions or uncertainty on how to do this, then I am happy to help. 

I asked KT about me doing it but he was adamant Hamish had to do the merge. 

When doing the Merge make sure to follow the instructions carefully and take screenshots of
EVERYTHING!

After this is done, I need to update the README file in the development branch to document the changes before 
creating and merging with the release branch and then back to the master branch.

Hamish's work in creating the TripCalculator branch and merging with the de

**Comment by 000961892** [comment]
I've merged TripCalculator into the development branch

**Reply by StephenHillenbrand** [reply]
Great new Hamish, hope it all went smooth
